app/views/returns/credits/ClaimForWhichYearView.scala.html

```diff
@@ -33,23 +35,27 @@
 
     @formHelper(action = controllers.returns.credits.routes.ClaimForWhichYearController.onSubmit, 'autoComplete -> "off") {
 
-        @heading(messages("claim-for-which-year.credits.title-heading"))
-
-        @paragraphBody(messages("claim-for-which-year.credits.paragraph.1"))
-
-        @govukRadios(
-            RadiosViewModel.apply(
-                field = form("value"),
-                legend = LegendViewModel(HtmlContent(hiddenContent("claim-for-which-year.credits.title-heading"))),
-                items = Seq(
-                    RadioItem(id = Some("value"), value = Some("true"), content =  Text("1 April 2022 to 31 March 2023"))
-                )
+        @if(form.errors.nonEmpty) {
```

Comment:
Would this way include the header in the error message (which I think we do not want) when we have a validation error?

Reply:
Might do, yeah. Will need to test that.